app/modules/asset/service.server.ts

```diff
@@ -244,6 +246,12 @@ export async function getAssets({
             },
             custody: {
               select: {
+                templateSigned: true,
```

Comment:
I am not sure why we need that in `getAssets`.  Are we showing this data anywhere on an index?

Reply:
I'm not entirely sure I get that question. If you mean the `templateSigned` field or the custody, I am using that to display the "SIgnature pending" logic

Reply:
@DonKoko please review this logic once so that I can mark its status

Reply:
@rajdip-b `getAssets` is used in the index as well as some other places. Including the templateSigned in all of those will require an extra join and will make the query slower in places where we dont use the data. We need to either allow getAssets & getPaginatedAndFiltered assets to receive a dynamic include or we need to make a separate function for this one case.